### ISSUE TITLE
double-clicking pyinstaller Mac app crashes in TrackPoll

### ISSUE THREAD

**Issue by aw-was-here** [aw-was-here/Now-Playing-Serato#28]
I've got it tracked down to the template code.  Opening the app package and running the SeratoNowPlaying binary or from the shell works.  Definitely odd!

**Comment by aw-was-here** [comment]
OK, I've tracked this down to Jinja failing to decode things.  This issue may be a blocker for using Jinja and will have to switch to something else. :frog:

**Comment by aw-was-here** [comment]
Tracked it down.

From python3 [docs](https://docs.python.org/3/library/functions.html#open):

```
encoding is the name of the encoding used to decode or encode the file. This should only be used in text mode. The default encoding is platform dependent (whatever locale.getpreferredencoding() returns), but any text encoding supported by Python can be used. See the codecs module for the list of supported encodings.
```

On OS X, when double clicking, it looks like the default encoding is something oddball that doesn't support the character sets I was trying to write.  Forcing the output to be UTF-8 works.  Ideally, this encoding should probably be user-settable but for now this fixes the crash.

**Comment by aw-was-here** [comment]
Fixed by #29
